pkg/river/token/builder/builder.go

```diff
@@ -131,6 +131,18 @@ func (b *Body) AppendFrom(goValue interface{}) {
 	b.encodeFields(rv)
 }
 
+// GetBlockLabel returns the label for a given block.
+func GetBlockLabel(rv reflect.Value) string {
+	tags := rivertags.Get(rv.Type())
+	for _, tag := range tags {
+		if tag.Flags&rivertags.FlagLabel != 0 {
+			return rv.FieldByIndex(tag.Index).String()
+		}
+	}
+
+	return ""
+}
```

Comment:
I'd prefer not to make this part of the public API for token/builder since it feels out of scope for the purpose of the package. Can we just make a copy of the unexported function or move this into river/internal/value or something?

Reply:
Will make a copy.